services/web/server/tests/unit/with_dbs/02/test_projects_handlers__open_close.py

```diff
@@ -480,7 +480,7 @@ async def test_open_project_with_small_amount_of_dynamic_services_starts_them_au
     all_service_uuids = list(project["workbench"])
     for num_service_already_running in range(num_of_dyn_services):
         mocked_director_v2_api["director_v2_api.list_dynamic_services"].return_value = [
-            {"service_uuid": all_service_uuids[service_id]}
+            {"service_uuid": all_service_uuids[service_id], "service_state": "running"}
```

Comment:
Question: why do I see the service_state being added here?

Reply:
In the code, we expect that "service_state" is there as we are filtering on him, it was missing in the unit tests when mocking this function.